www/docusaurus.config.js

```diff
@@ -69,7 +69,13 @@ const config = {
           {
             position: 'left',
             label: 'Guides',
-            to: '/guides/intro',
+            to: '/docs/guides/intro',
```

Comment:
I recommend replacing the `to` property in favour of `type: 'doc'` and `docId: 'guides/intro'` like the preceding `API` entry so that the version name infix is preserved in the navbar. 

To clarify, by going to the [preview](https://deploy-preview-505--starknetjs.netlify.app/docs/next/API/) with the `Next` version selected so that the `next` infix is present in the path and looking at the `API` and `Guides` navigation links it can be seen that their paths are:
- API: `/docs/next/API/`
- Guides: `/docs/guides/intro`

The recommended change should make it so that the `Guides` path is the following:
- Guides: `/docs/next/guides/intro`

---

I also noticed a similar issue in the footer, however, that seems to be a missing functionality in Docusaurus itself.

Reply:
Done.